test/scenarios/re_emissions_with_thermal.json

```diff
@@ -95,7 +95,6 @@
     "min_kw": 200.0,
     "min_allowable_kw":0.0,
     "max_kw": 200.0,
-    "can_supply_steam_turbine": true,
```

Comment:
was this removed until the steam turbine emissions TODO (that I think exists) is addressed?

Reply:
I think it was just removed for debugging purposes before I had implemented SteamTurbine and uncommented that field in CHP. Since there is no SteamTurbine in this test yet, that field won't do anything.